template-files/action.yml

```
@@ -0,0 +1,38 @@
+---
+name: Template Files
+description: Template (or copy) files from other repositories and commits them to the specified PR.
+author: Anaconda Inc.
+branding:
+  icon: book-open
+  color: green
+
+inputs:
+  config:
+    description: Configuration path defining what files to template/copy.
+    default: .github/templates/config.yml
```

I would suggest naming this like the action.
```suggestion
    default: .github/template-files/config.yml
```